kyuubi-util-scala/src/main/scala/org/apache/kyuubi/util/reflect/ReflectUtils.scala

```diff
@@ -92,14 +88,8 @@ object ReflectUtils {
       }
     } catch {
       case e: Exception =>
-        val candidates =
-          (clz.getDeclaredMethods ++ clz.getMethods)
-            .map(m => s"${m.getName}(${m.getParameterTypes.map(_.getName).mkString(", ")})")
-            .distinct.sorted
-        val argClassesNames = argClasses.map(_.getName)
         throw new RuntimeException(
-          s"Method $methodName(${argClassesNames.mkString(", ")})" +
-            s" not found in $clz [${candidates.mkString(", ")}]",
+          s"$clz does not have $methodName method or incorrect arguments",
```

Comment:
I suggest to keep the method signature `$methodName(${argClassesNames.mkString(", ")})` for further investigation. As for skipping candidates, LGTM.